sql/catalyst/src/main/scala/org/apache/spark/sql/internal/connector/SupportsPushDownCatalystFilters.scala

```diff
@@ -35,7 +35,7 @@ trait SupportsPushDownCatalystFilters {
 
   /**
    * Returns the data filters that are pushed to the data source via
-   * {@link #pushFilters(Expression[])}.
+   * {@link #pushFilters(Predicate[])}.
    */
-  def pushedFilters: Array[Filter]
+  def pushedFilters: Array[Predicate]
```

Comment:
This is introduced in 3.3. I think we need to change this to V2Filter in this release so it won't break in the future.